x/ibc/02-client/types/codec.go

```diff
@@ -90,3 +90,41 @@ func UnpackConsensusState(any *codectypes.Any) (exported.ConsensusState, error)
 
 	return consensusState, nil
 }
+
+// PackClientState constructs a new Any packed with the given client state value. It returns
+// an error if the client state can't be casted to a protobuf message or if the concrete
+// implemention is not registered to the protobuf codec.
+func PackClientState(clientState exported.ClientState) (*codectypes.Any, error) {
```

Comment:
can you add a test case for each of these funcs?

Reply:
also code ordering wise I think putting these above consensus states makes sense